app/src/test/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModelTest.kt

```diff
@@ -25,5 +31,12 @@ class ProfileViewModelTest : ShouldSpec(
 
 			coVerify { authRepository.deleteAccount() }
 		}
+
+		should("profile return Profile data WHEN ProfileViewModelTest was initialized") {
+			val profile = mockk<Profile>()
+			coEvery { authRepository.userProfile() } returns profile
```

Comment:
coEvery probably is not needed anymore as we don't have coroutines here :D

Reply:
Yes, it is not needed, I deleted it 😄